script/env.sh

```diff
@@ -11,12 +11,19 @@ npx --yes playwright install-deps
 #-------------
 printf "\n\n Installing required python packages...."
 sudo apt install -y python3-pip
-sudo -H pip3 install mkdocs
-sudo -H pip3 install mkdocs-material
-sudo -H pip3 install python-markdown-math
-sudo -H pip3 install mkdocs-open-in-new-tab
-sudo -H pip3 install mkdocs-with-pdf
-sudo -H pip3 install qrcode
+
+python3 -m venv ./dtaas-venv
```

Comment:
please update the [developer.sh](https://github.com/INTO-CPS-Association/DTaaS/blob/feature/distributed-demo/deploy/vagrant/make_boxes/dtaas/developer.sh) as well.